src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,136 @@
++++
+title = "Troubleshooting Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 5
+pre = "<b>1. </b>"
++++
+
+## Overview
+
+You have folowed steps in [Deployment](../deployment) but something has gone wrong. You're not sure what and how to fix it, or what information to collect to raise an issue. Welcome to the Submariner troubleshooting guide where we will help you get your deployment working again.
+
+To recap, Submariner consists of two main core components (broker and submariner) and optional components (lighthouse), detailed in [Architecture](../architecture) section. Basic familiarity with the architecture will be helpful when troubleshooting.
+
+The guide has been broken into different sections for easy navigation.
+
+### Pre-requisite
+Before we begin troubleshooting, it would be good to know which version of submariner and its components you are running.
+
+{{% notice info %}}
+
+TBD: Add information on how to find out which version users are running. Is `subctl version` enough?
```

Comment:
subctl version only shows the version of the subctl binary, but that'd be a cool improvement.

We can look at the operator status, may be we can add also that information into subctl "version" -> looks the operator status, and returns those details too?

Reply:
I'll raise an issue to add more useful information to `subctl version`